AboutDoc/Documentation_About.rst

```diff
@@ -0,0 +1,250 @@
+.. Copyright (c) 2020-2020 OpenShot Studios, LLC
+ (http://www.openshotstudios.com).
+ This file is part of OpenShot Video Editor (http://www.openshot.org),
+ an open-source project dedicated to delivering high quality video editing and animation solutions to the world.
+
+.. OpenShot Video Editor is free software:
+ you can redistribute it and/or modify it under the terms of the GNU General Public License as published by  the Free Software Foundation, 
+ either version 3 of the License,
+ or (at your option) any later version.
+
+.. OpenShot Video Editor is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ See the GNU General Public License for more details.
+
+.. You should have received a copy of the GNU General Public License
```

Comment:
The standard heading shouldn't be reformatted, the copyright years can be updated but other than that it shouldn't really be messed with as it's in part a license statement. Consistency with other files / history is more important than consistency with the rest of the file's formatting.

```suggestion
.. Copyright (c) 2008-2020 OpenShot Studios, LLC
 (http://www.openshotstudios.com). This file is part of
 OpenShot Video Editor (http://www.openshot.org), an open-source project
 dedicated to delivering high quality video editing and animation solutions
 to the world.

.. OpenShot Video Editor is free software: you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation, either version 3 of the License, or
 (at your option) any later version.

.. OpenShot Video Editor is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.

.. You should have received a copy of the GNU General Public License
 along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
```